frontend/src/Calendar/Calendar.module.css

```diff
@@ -0,0 +1,58 @@
+.calendarPage {
+    height: 100%;
+    gap: 16px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.calendarElem {
+    height: 90vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+.test{
+    display: flex
```

Comment:
don't forget `;` - maybe double check prettier is formatting css files? there seems to be a couple inconsistencies thru the file